### ISSUE TITLE
Updated character sheets

### ISSUE THREAD

**Issue by Lochabar** [chummer5a/chummer5a#1810]
Here is a complete replacement of the sheets sub-folder.

[sheets.zip](https://github.com/chummer5a/chummer5a/files/1156909/sheets.zip)

Changes include
1) Further simplifying the character viewer sheets
- removed the Includes to the "set" members
- replaced individual language file names (e.g. xz.de or xz.fr) with a specific 
"xz.language" file in each language folder

2) Corrected names of French and German character sheets following changes in issue #1541.
German
- Schick Blöcke [not Schick Blîcke]
- Shadowrun 5 (Bewertet größer 0) [not Shadowrun 5 (Bewertet grî·er 0)]
- Shadowrun 5 (Gruppiert Fähigkeit durch Bewertet) [not Shadowrun 5 (Gruppiert 
FÑhigkeit durch Bewertet)]
- Shadowrun 5 (Gruppiert Fähigkeit durch Name) [not Shadowrun 5 (Gruppiert FÑhigkeit 
durch Name)]
- Spielleiter Überblick [not Spielleiter öberblick]
French
- Dépenses [not DÇpenses]
- Formaté texte seulement [not FormatÇ texte seulement]
- Shadowrun 5 (Évaluation plus grand que zéro) [not Shadowrun 5 (êvaluation plus 
grand que zÇro)]
- Shadowrun 5 (Groupé par évaluation) [not Shadowrun 5 (GroupÇ par Çvaluation)]
- Shadowrun 5 (Groupé par nom) [not Shadowrun 5 (GroupÇ par nom)]
- Véhicules [not VÇhicules]

3) Added prints of Locations to Text-Only sheet as requested in issue #1541. Applied same changes to Formatted Text-Only sheet.

4) Includes fix for #1790 to print the spell category correctly when using other than English.

**Comment by Lochabar** [comment]
I provided a complete replacement for sheets.

The files that were wrongly names (listed above), that were provided with the correct name, have been renamed again (e.g. Schick Blöcke is once again Schick Blîcke). Is this a GitHub problem?



Files that were added are missing:
sheets/xz.language.xslt
sheets/de/xz.language.xslt
sheets/fr/xz.language.xslt
sheets/jp/xz.language.xslt
Note: these are language specific files not copies.


And files that were no removed are still present:
sheets/Shadowrun 5 includes.xslt
sheets/xz.en-us.xslt

sheets/de/Shadowrun 5 includes.xslt
sheets/de/xz.de.xslt

sheets/fr/Shadowrun 5 includes.xslt
sheets/fr/xz.de.xslt

sheets/jp/Shadowrun 5 includes.xslt
sheets/jp/xz.de.xslt
